File: DotNet/CesiumLanguageWriter/Advanced/ICesiumColorBlendModeValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a color blend mode.
    /// </summary>
    public interface ICesiumColorBlendModeValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value expressed as a color blend mode.
        /// </summary>
        /// <param name="value">The blend mode.</param>
        void WriteColorBlendMode(CesiumColorBlendMode value);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumColorBlendModeValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;
import javax.annotation.Nonnull;

/**
 * A writer that can write a value as a color blend mode.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumColorBlendModeValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value expressed as a color blend mode.
    * @param value The blend mode.
    */
    void writeColorBlendMode(@Nonnull CesiumColorBlendMode value);
}